src/UAEPass/Provider.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+namespace SocialiteProviders\UAEPass;
+
+use GuzzleHttp\RequestOptions;
+use Illuminate\Support\Arr;
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+/**
+ * @see https://docs.uaepass.ae/guides/authentication/web-application
+ */
+class Provider extends AbstractProvider
+{
+    public const IDENTIFIER = 'UAEPASS';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected $scopes = ['urn:uae:digitalid:profile:general'];
```

Comment:
are these 5 different scopes? if so, can we have them as single keys, not pre-joined with the scope sep?

Reply:
Updated